Brocfile.js

```diff
@@ -123,7 +123,11 @@ for (var packageName in packages.dependencies) {
   trees.push(pickedEs6Lib);
 
   // AMD lib
-  var transpiledAmdLib = transpileES6(libTree, { moduleName: true, type: 'amd' });
+  var transpiledAmdLib = new ES6Modules(libTree, {
+    moduleName: true,
+    type: 'amd',
+    strict: true,
```

Comment:
Should be in `esperanoOptions`.  See https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L654-L661.

Also, you have to specify the `_evilES3SafeReExports` option (ugh, I know) for IE8 support.